docs/code-snippets/change-a-currency-symbol.md

```diff
@@ -3,9 +3,9 @@ post_title: Change a currency symbol
 tags: code-snippet
 ---
 
-See the [currency list](https://woocommerce.github.io/code-reference/files/woocommerce-includes-wc-core-functions.html#source-view.475) for reference on currency codes.
+In WooCommerce, each currency is associated with a code and a symbol. For example, the Australian dollar has the code `AUD` and the symbol `$` (if you are interested, you can see a full list of codes and symbols in the [source code](https://woocommerce.github.io/code-reference/files/woocommerce-includes-wc-core-functions.html#source-view.662)). 
```

Comment:
I updated the link, because it seemed to be pointing to the wrong section of the source ... but, the line number will probably continue to fall out of sync with this reference. Might it be better to link to a specific commit or tag, such as:

https://github.com/woocommerce/woocommerce/blob/9.6.1/plugins/woocommerce/includes/wc-core-functions.php#L682

Or, to re-word my question, is it a requirement/expectation to use https://woocommerce.github.io/code-reference/ when linking to the source?

Reply:
```suggestion
In WooCommerce, each currency is associated with a code and a symbol. For example, the Australian dollar has the code `AUD` and the symbol `$` (if you are interested, you can see a full list of codes and symbols in the [source code](https://github.com/woocommerce/woocommerce/blob/9.6.1/plugins/woocommerce/includes/wc-core-functions.php#L682)). 
```